pnpm-workspace.yaml

```diff
@@ -0,0 +1,3 @@
+packages:
+  - "packages/nextjs"
+  - "packages/hardhat"
```

Comment:
Could this also be `'packages/*'`? Thinking about the cli :D

Reply:
Yup fixed at https://github.com/scaffold-eth/scaffold-eth-2/pull/444/commits/ba393c0061a698343590b66ae34994ed13a4ae81 🙌